pose_prediction/const_accel/plugin.cpp

```diff
@@ -210,10 +205,9 @@ class pose_prediction_impl : public pose_prediction {
     switchboard::reader<switchboard::event_wrapper<time_point>> _m_vsync_estimate;
 	mutable Eigen::Quaternionf offset {Eigen::Quaternionf::Identity()};
 	mutable std::shared_mutex offset_mutex;
-    
 
     // Slightly modified copy of OpenVINS method found in propagator.cpp
-    // Returns a pair of the predictor state_plus and the time associated with the
+    // Returns a pair of the predictor state and the time associated with the
     // most recent imu reading used to perform this prediction.
     std::pair<Eigen::Matrix<double,13,1>,time_point> predict_mean_rk4(double dt) const {
```

Comment:
Consider change the function name to `predict_const_accel` to be consistent with `predict_const_vel`.